python/cuml/ensemble/randomforestregressor.pyx

```diff
@@ -499,16 +497,14 @@ class RandomForestRegressor(Base):
                 <ModelHandle> mod_ptr))
 
         concat_model_handle = concatenate_trees(deref(model_handles))
-
-        concat_model_ptr = <size_t> concat_model_handle
-        self.concat_handle = ctypes.c_void_p(concat_model_ptr).value
-        cdef uintptr_t model_ptr = <uintptr_t> self.concat_handle
+        cdef uintptr_t concat_model_ptr = <uintptr_t> concat_model_handle
```

Comment:
I think these internal functions around concatenation all could benefit from short docstrings as they can be non-obvious otherwise.